File: GomokuServer/src/GomokuServer.Application/Games/Responses/GetActiveGamesResponse.cs
namespace GomokuServer.Application.Games.Responses;

public record GetActiveGamesResponse(string GameId)
{
	[Required]
	public string GameId { get; init; } = GameId;

	public PlayersDto? Players { get; init; }

	public TimeControlDto? TimeControl { get; init; }
}

File: GomokuClient/packages/gomoku-api/client/models/GetActiveGamesResponse.ts
import type { PlayersDto } from "./PlayersDto";
import type { TimeControlDto } from "./TimeControlDto";

 export type GetActiveGamesResponse = {
    /**
     * @type string
    */
    gameId: string;
    /**
     * @type object | undefined
    */
    players?: PlayersDto;
    /**
     * @type object | undefined
    */
    timeControl?: TimeControlDto;
};